docs/admin/Offline_Cert_Renewal.md

```diff
@@ -0,0 +1,178 @@
+The CS.cfg is located in `/etc/pki/<instance>/<subsystem>/CS.cfg`
+
+    ````
+    selftests.container.order.startup=CAPresence:critical, SystemCertsVerification:critical
+    ````
+
+    **OR**
+
+    Use the built-in tool:
+
+    ````
+    # pki-server selftest-disable -i <instance_name>
+    ````
+2. Synchronize NSS DB and CS.cfg
+
+    ````
+    # pki-server cert-update <cert ID> # for all system certificates that is to be renewed
+    ````
+
+### Bringing up the PKI server
+
+1. Create temp SSL certificate
+    ````
+    # pki-server cert-create sslserver --temp
+    ````
+
+2. Delete the existing System cert if exist
+    ````
+    # pki-server cert-del sslserver
+    ````
+
+3. Import temp SSL certificate created in previous step
+    ````
+    # pki-server cert-import sslserver
+    ````
+
+4. Start server
+    ````
+    # systemctl start pki-tomcatd@pki-tomcat
+    ````
+
+### System Certificate Renewal
+
+1. Renew required system certs using PKI tool:
+    ````
+    # pki-server cert-create --renew \
+    -n <admin nickname> \
+    -d <NSS db path> \
+    -c <NSS client DB password>
+    <cert ID> 
+    --serial <serial No.> # Provide it from SSL server cert to avoid placing request for unintended cert
+    ````
+    **OR**
+
+    using 3rd party tool (like certmonger). **Skip to step #4 after this step, if using this option**.
+    ````
+    # getcert list
+    # getcert resubmit -i <id> # Get the ID of the tracked cert from the previous command
+    ````
+
+2. Stop server to update PKI server instance to use latest renewed certs
+    ````
+    # systemctl stop pki-tomcatd@pki-tomcat
+    ````
+
+3. Delete the existing NSS db certs
+    ````
+    # pki-server cert-del <cert ID>
+    ````
+
+4. Import the renewed permanent certs into NSS db and update corresponding CS.cfg files
+    ````
+    # pki-server cert-import <cert ID>
+    ````
+
+    *Note:* Make sure the **Audit Log** has the trust flags: ***"u,u,Pu"*** by running the following command:
+    ````
+    # certutil -L -d /var/lib/pki/pki-tomcat/alias/
+
+    Certificate Nickname                                         Trust Attributes
+                                                                 SSL,S/MIME,JAR/XPI
+    ca_signing                                                   CTu,Cu,Cu
+    transportCert cert-pki-tomcat KRA                            u,u,u
+    storageCert cert-pki-tomcat KRA                              u,u,u
+    auditSigningCert cert-pki-tomcat KRA                         u,u,Pu
+    subsystem                                                    u,u,u
+    sslserver                                                    u,u,u
+    ca_ocsp_signing                                              u,u,u
+    ca_audit_signing                                             u,u,Pu
+    ````
+5. Enable the self test. Add the following highlighted line CS.cfg of the corresponding subsystem
```

Comment:
Remove the instruction to manually edit the CS.cfg. It's no longer necessary since we already have the pki-server selftest-enable command.